modules/crop/architecture.md

```diff
@@ -0,0 +1,47 @@
+# Architecture
+
+Diagram shows basic architecture of Crop.
+
+![Crop architecture](images/architecture.png)
+
+## Components
+
+### Entity
+- **Crop** Is a content entity that stores various informations related to the crop area.
+- **CropType** CropType is bundle for Crop entity.
+
+###Plugin
+- **CropEffect** Crop effect is an image style effect. He apply your crop area into image with information of Crop entity.
+
+This diagram shows how specifically Crop Image Effect (Manual crop) get informations of corresponding image ressource and apply different effect.
```

Comment:
I think it would be good to clarify how Crop entity is used by imageStyle and how the recovery of crop has applied to the resource. It seems to me the scope Crop entity. I have not made voluntarily detail the portion not off topic.